src/monster.cpp

```diff
@@ -1309,15 +1309,52 @@ void monster::add_effect( efftype_id eff_id, int dur, body_part bp,
 
 int monster::get_armor_cut(body_part bp) const
 {
-    (void) bp;
+    (void)bp;
     // TODO: Add support for worn armor?
-    return int(type->armor_cut) + armor_bash_bonus;
+    return int(type->armor_cut) + get_armor_cut_bonus( bp );
 }
 
-int monster::get_armor_bash(body_part bp) const
+int monster::get_armor_bash( body_part bp ) const {
+    (void)bp;
```

Comment:
Same here. the line exists to avoid warnings "the parameter bp is not used" from the compiler. If it is used, the line is pointless.